BoxLEDs.cpp

```diff
@@ -40,18 +40,18 @@ void BoxLEDs::_handleAnimation(ANIMATION* animation) {
         setAll(_wheel(random(255)));
     } else if (animation->type == ANIMATION_TYPE::PULSE) {
         if (animation->direction == ANIMATION_DIRECTION::UP) {
-            if (animation->state <= LED_PWM_MAX - animation->step) {
+            if (animation->state <= 0xFF - animation->step) {
                 animation->state = animation->state+animation->step;
```

Comment:
```suggestion
                animation->state += animation->step;
```

Reply:
Resolved in 837300d9fd0072863f806b5fe18a7aa8006ec632